docs/migrating-from-onenote.md

```diff
@@ -2,19 +2,32 @@
 
 This guide mostly duplicates the instructions at the repo for the PowerShell [script](https://github.com/nixsee/ConvertOneNote2MarkDown).
 
+## Summary
+
+The powershell script 'ConvertOneNote2MarkDown-v2.ps1' will utilize the OneNote Object Model on your workstation to convert all OneNote pages to Word documents and then utilizes PanDoc to convert the Word documents to Markdown (.md) format. It will also:
+
+* Create a folder structure for your Notebooks and Sections.
+* Process pages that are in sections at the Notebook, Section Group and 1st Nested Section Group levels.
+* Allow you you choose between putting all Images in a central '/media' folder for each notebook, or in a separate '/media' folder in each folder of the hierarchy.
+* Fix image references in the resulting .md files, generating relative references to the image files within the markdown document.
+* A title, description, and date header will be added to each file as well.
+* And more (see details at repo)!
+
 ## Usage
 
-1. Start the OneNote application. All notebooks currently loaded in OneNote will be converted
-2. It is advised that you install Onetastic and the attached macro, which will automatically expand any collapsed paragraphs in the notebook. They won't be exported otherwise.
+1. Start the OneNote application. All notebooks currently loaded in OneNote will be converted.
+2. It is advised that you install [Onetastic](https://getonetastic.com/download) and the attached macro, which will automatically expand any collapsed paragraphs in the notebook. They won't be exported otherwise.
     * To install the macro, click the New Macro Button within the Onetastic Toolbar and then select File -> Import and select the .xml macro included in the release.
     * Run the macro for each Notebook that is open
 3. It is highly recommended that you use VS Code, and its embedded Powershell terminal, as this allows you to edit and run the script, as well as check the results of the .md output all in one window.
-4. Whatever you choose, open a PowerShell terminal and navigate to the folder containing the script and run it.
+4. Whatever you choose, open a PowerShell terminal and navigate to the folder containing the script (use `cd` to navigate to the right directory). Run the script by executing
```

Comment:
I think this is still unclear for the general user. Could also include a "right click on folder in VS Code to open terminal at that path" sort of instruction. (i should add these to the instructions in the original repo as well)

Reply:
I've added more detail here.